api/src/main/resources/underlays/sd.yaml

```diff
@@ -0,0 +1,2604 @@
+# An initial OMOP underlay schema for development.
+
+name: sd
+entities:
+  - name: person
```

Comment:
As a self-reminder for my next pass on this PR, I want to diff this against the aou synthetic underlay locally